public/components/overview/home.tsx

```diff
@@ -11,60 +19,217 @@ import {
 
 export const Home = () => {
   const [homePage, setHomePage] = useState<ReactNode>(<></>);
   const [dashboardsSavedObjects, setDashboardsSavedObjects] = useState<DashboardSavedObjectsType>(
     {}
   );
   const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [showGetStarted, setShowGetStarted] = useState<boolean | null>(null); // Initial null state
+
   ObsDashboardStateManager.showFlyout$.next(() => () => setIsFlyoutVisible(true));
 
   const loadHomePage = () => {
     setHomePage(coreRefs.contentManagement?.renderPage(HOME_PAGE_ID));
   };
 
-  const registerCards = async () => {
-    let alertsPluginExists = false;
-    let anomalyPluginExists = false;
-    try {
-      const res = await coreRefs.http?.get('/api/status');
-      if (res) {
-        for (const status of res.status.statuses) {
-          if (status.id.includes(alertsPluginID)) {
-            alertsPluginExists = true;
-          }
-          if (status.id.includes(anomalyPluginID)) {
-            anomalyPluginExists = true;
-          }
+  const togglePopover = () => setIsPopoverOpen(!isPopoverOpen);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  const loadDashboardState = () => {
+    coreRefs.savedObjectsClient
+      ?.find({
+        type: 'dashboard',
+      })
+      .then((response) => {
+        const savedDashboards = response.savedObjects.reduce((acc, savedDashboard) => {
+          const dashboardAttributes = savedDashboard.attributes as {
+            title: string;
+            timeFrom: string;
+            timeTo: string;
+          };
+          const id = savedDashboard.id.toString();
+          acc[id] = {
+            value: id,
+            label: dashboardAttributes.title,
+            startDate: dashboardAttributes.timeFrom,
+            endDate: dashboardAttributes.timeTo,
+          };
+          return acc;
+        }, {} as DashboardSavedObjectsType);
+
+        setDashboardsSavedObjects(savedDashboards);
+        const defaultDashboard = getObservabilityDashboardsId();
+
+        if (defaultDashboard in savedDashboards) {
+          ObsDashboardStateManager.dashboardState$.next({
+            dashboardTitle: savedDashboards[defaultDashboard].label,
+            dashboardId: defaultDashboard,
+            startDate: savedDashboards[defaultDashboard].startDate ?? 'now-7d',
+            endDate: savedDashboards[defaultDashboard].endDate ?? 'now',
+          });
+          ObsDashboardStateManager.isDashboardSelected$.next(true);
+        } else {
+          setObservabilityDashboardsId(null);
+          ObsDashboardStateManager.dashboardState$.next({
+            startDate: '',
+            endDate: '',
+            dashboardTitle: '',
+            dashboardId: '',
+          });
+          ObsDashboardStateManager.isDashboardSelected$.next(false);
         }
+      })
+      .catch((error) => {
+        console.error('Error fetching dashboards:', error);
```

Comment:
should this be a console error? I know in before we seems to have some console errors on these fetch failures. I think it is okay to leave it here for now, but maybe we can have some better error handling as a follow up.

Reply:
Will look into, thanks for the call out.